### DISCUSSION TITLE
Configure plustoolkit for usb video camera or any capture card on windows

### DISCUSSION THREAD

**Discussion by Gowtham1901** [PlusToolkit/PlusLib#927] [Q&A]
Hi, 
I am trying to set up plustoolkit to stream video from windows capture card to OpenIGTlink server. I am fairly new to this so I am not able to get the user guide in plustoolkit website. 
Can anyone help me in how to get started?

**Comment by Sunderlandkyl** [comment]
If your device is compatible with the Microsoft Media Foundation interface, you can use this device: http://perk-software.cs.queensu.ca/plus/doc/nightly/user/DeviceMicrosoftMediaFoundation.html

Try "PlusServer: Media Foundation video capture device - color", or "PlusServer: Media Foundation video capture device" from the device set selector.

**Reply by Gowtham1901** [reply]
Thanks for the quick response @Sunderlandkyl, will tryout Microsoft MF compatibility and check again.
